src/core/Form/Checkbox/Checkbox.baseStyles.tsx

```diff
@@ -57,6 +57,7 @@ const largeVariantStyles = (theme: SuomifiTheme) => css`
     & .fi-checkbox_label {
       padding-left: ${theme.spacing.xxl};
       min-height: 30px;
```

Comment:
Does the min-height style serve a purpose now that the line-height is set to 30 as well?

Reply:
Same goes for the smaller variant.

Reply:
Seems that there is no effect with min-height or without. So I removed it now.